feat: add shadcn UI components and create projects overview page

mochiquin opened the pull request:
- Add new shadcn/ui components: badge, dropdown-menu, logo, pagination, placeholder-logo, progress,   separator, table
 - Update button component with improved styling and  accessibility
  - Crate new projects page (/projects) for post-login  project overview
  - Implement GitHub repository import interface with input field and import button
  - Add project listing table with columns for name,last updated, status, and repository URL
  - Set up proper routing structure for projects section

mochiquin committed:
feat: add shadcn UI components and create projects overview page
  - Add new shadcn/ui components: badge, dropdown-menu, logo, pagination, placeholder-logo, progress,   separator, table
 - Update button component with improved styling and  accessibility
  - Crate new projects page (/projects) for post-login  project overview
  - Implement GitHub repository import interface with input field and import button
  - Add project listing table with columns for name,last updated, status, and repository URL
  - Set up proper routing structure for projects section

mochiquin committed:
feat: add shadcn UI components and create projects overview page
  - Add new shadcn/ui components: badge, dropdown-menu, logo, pagination, placeholder-logo, progress,   separator, table
 - Update button component with improved styling and  accessibility
  - Crate new projects page (/projects) for post-login  project overview
  - Implement GitHub repository import interface with input field and import button
  - Add project listing table with columns for name,last updated, status, and repository URL
  - Set up proper routing structure for projects section

mochiquin committed:
feat(profile): implement profile page with navigation and settings options; add select component for enhanced UI

mochiquin committed:
feat(profile): implement profile page with navigation and settings options; add select component for enhanced UI

CarlosFuSiCong commented on frontend/components/auth/AuthSidebar.tsx:
你尝试把<P>里的内容 单独提出来 一个变量

mochiquin replied:
alreadly fix it

CarlosFuSiCong committed:
refactor(projects): enhance project model and API

- Add soft delete support with deleted_at field
- Add repo_type field for different Git providers
- Add description field for project details
- Add risk assessment fields (stc_risk_score, mcstc_risk_score)
- Update API to support filtering and sorting
- Remove unnecessary status fields and configurations

mochiquin committed:
Update AuthSidebar.tsx

Fix and add a new Variable to change the text

mochiquin committed:
Merge branch 'feature/projects-page' of https://github.com/CarlosFuSiCong/Secuflow_Rebuild into feature/projects-page

mochiquin committed:
Update AuthSidebar.tsx

fix the Variable of text
